tests/func/test_run.py

```diff
@@ -323,11 +322,6 @@ def test_run_args_with_spaces(self):
         self.assertEqual(ret, 0)
         self.assertEqual(stage.cmd, 'echo "foo bar"')
 
-    @mock.patch.object(subprocess, "Popen", side_effect=KeyboardInterrupt)
```

Comment:
I think we can instead check that communicate has been called on keyboard interrupt

Reply:
Actually, I'm not sure how to do that)
Also, I think we may take Popen object creation out of try-except statement and leave only
```
try:
    p.communicate()
except KeyboardInterrupt:
    p.communicate()
```
With this approach old test will work (if we need it) and the only thing that is left is to come up with a test to check that communicate has been called on keyboard interrupt)